packages/admin/admin-theme/src/typographyOptions.ts

```diff
@@ -3,12 +3,18 @@ import { TypographyOptions } from "@mui/material/styles/createTypography";
 
 const fontFamily = "Roboto, Helvetica, Arial, sans-serif";
 
+// https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#common_weight_name_mapping
 export const fontWeights = {
-    fontWeightLight: 100,
-    fontWeightSemiLight: 200,
-    fontWeightRegular: 300,
-    fontWeightMedium: 400,
-    fontWeightBold: 500,
+    thin: 100,
+    extraLight: 200,
+    light: 300,
+    regular: 400,
+    medium: 500,
+    semiBold: 600,
+    bold: 700,
+    extraBold: 800,
+    black: 900,
+    extraBlack: 950,
```

Comment:
@jamesricky Thanks for the heads-up

I removed `extraBold`, `black` and `extraBlack` as I couldn't find them in the designs.

The new font weights can be used out-of-the-box because the system is based on simple pattern matching:

```tsx
<Typography fontWeight="thin">Thin</Typography>
<Typography fontWeight="extraLight">Extra Light</Typography>
<Typography fontWeight="light">Light</Typography>
<Typography fontWeight="regular">Regular</Typography>
<Typography fontWeight="medium">Medium</Typography>
<Typography fontWeight="semiBold">Semi Bold</Typography>
<Typography fontWeight="bold">Bold</Typography>
```

<img width="122" alt="Bildschirmfoto 2024-04-09 um 16 09 07" src="https://github.com/vivid-planet/comet/assets/13380047/f330f66d-cd66-4966-a39e-c84b1924a7e2">

There are no explicit types for the fontWeights (so there is also no type that must be extended).

---

I encountered a problem with the fontsource imports because it seems like the package doesn't have all weights. These are the files from the package:

<img width="181" alt="Bildschirmfoto 2024-04-09 um 15 45 49" src="https://github.com/vivid-planet/comet/assets/13380047/a62690f8-d925-4c35-9321-0201d0cb4390">

200 and 600 seem to be missing although we need them. Am I doing something wrong or is that okay?

Reply:
> 200 and 600 seem to be missing although we need them. Am I doing something wrong or is that okay?

That means we are not actually loading/using that font... seems like Roboto doesn't have the those weight's in general 🤔

Reply:
> That means we are not actually loading/using that font... seems like Roboto doesn't have the those weight's in general 🤔

How much of a problem is that? Doesn't the browser handle that as a fallback by just making it bolder?

Reply:
> Doesn't the browser handle that as a fallback by just making it bolder?

I think it does, I'm not sure how reliable/accurate this is. 
Maybe it just happens to look fine on our machines 😅 

But if UX explicitly defines a text to be rendered as Roboto 200, we should also do that instead of letting the browser guess how it should look. 

I asked UX where they got the font from, assuming they actually have it, maybe we can use that.